codegen/templates/grpc_client.tmpl

```diff
@@ -76,8 +76,9 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 		circuitBreakerDisabled = deps.Default.Config.MustGetBoolean("clients.{{$clientID}}.circuitBreakerDisabled")
 	}
 	if !circuitBreakerDisabled {
-		for methodKey := range methodNames {
-			configureCircuitBreaker(deps, timeoutInMS, methodNames[methodKey])
+		for _, methodName := range methodNames {
+			circuitBreakerName := "{{$clientID}}"  + "-" + methodName
```

Comment:
Are we doing something different for grpc_client.tmpl and http_client.tmpl? http Client has methodKey vs here we are using methodName

Reply:
Here I'm doing methodName (the value of the map) because the map of methods has the method name as the value and the service method as the key. 
<img width="532" alt="Screen Shot 2021-07-14 at 9 28 08 AM" src="https://user-images.githubusercontent.com/13949280/125639383-14d4b59f-bd2f-4f03-aa7f-4d5abae86673.png">
But in http_client.tmpl I use the key value of the map because the map has the methodname as the value.
<img width="617" alt="Screen Shot 2021-07-14 at 9 27 00 AM" src="https://user-images.githubusercontent.com/13949280/125639188-3579cdd8-cbbb-41c6-8890-e0c7569e6aa9.png">
I could change the naming so both files have methodName as the variable in the loop, what do you think?

Reply:
Thanks for clarifying! That makes sense. Yeah - I think keeping it consistent across templates would improve readability.